app/xlist/scripts/HomeController.js

```diff
@@ -125,5 +125,25 @@ angular
       });
     };
 
+    $scope.congratsAlert = function(task) {
+      task.save({
+        done: !task.attributes.done
```

Comment:
Can we stick with using `task.get{'done')`, so we don't encounter any more weird problems accessing `attributes` directly.